build.xml

```diff
@@ -46,4 +46,13 @@
     <mkdir dir="${build.conf.dir}"/>
     <copy file="${tz.data}" todir="${build.conf.dir}"/>
   </target>
+
+  <target name="pkg" depends="timezones">
+    <delete dir="build/dist"/>
+    <delete dir="build/stage"/>
+    <exec dir="." executable="./pkg-builder.pl" failonerror="true">
+       <arg value="--define"/>
+       <arg value="jetty.distro=jetty-distribution-9.3.5.v20151012"/>
```

Comment:
This should be resolved in the next task which is jetty restructuring. We should not depend on hardcoded jetty versions sprinkled across. This is not the only place where jetty version is hard coded. All of them is planned to be addressed in the jetty restructuring task.